manifests/3.0.0-alpha1/opensearch-3.0.0-alpha1.yml

```diff
@@ -3,6 +3,7 @@ schema-version: '1.1'
 build:
   name: OpenSearch
   version: 3.0.0
+  qualifier: alpha1
```

Comment:
Should it be rc1? Looking at past releases https://opensearch.org/releases.html#release-history.
cc: @reta @dblock

Reply:
Hi @gaiksaya ,

No, we have proposed to have alpha1 and beta1 just like previously in 1.x/2.x.

The differences is, 1.x and 2.x have alpha1/beta1/rc1 before we come up with the RC concept for every release.

Therefore to avoid confusion we will have alpha1/beta1/GA(no qualifier) for 3.x.

https://github.com/opensearch-project/.github/issues/252

Thanks.

Reply:
Is there a possibility of `alpha2`? If not why not just move ahead with alpha/beta/GA?

Reply:
Because in core there is a rule in code, that qualifier has to be `char+number`, therefore alpha will not even allow core to be compiled and error our.

Reply:
This has already been communicated with Andrew and Saurabh.